paddle/operators/lrn_op.cc

```diff
@@ -19,6 +19,108 @@ namespace operators {
 
 using framework::Tensor;
 
+template <typename T>
+struct LRNFunctor<platform::CPUPlace, T> {
+  void operator()(const framework::ExecutionContext& ctx,
+                  const framework::Tensor* input, framework::Tensor* out,
```

Comment:
For input arguments: `const framework::Tensor& `

https://google.github.io/styleguide/cppguide.html#Reference_Arguments

Reply:
Done!
Thanks!